components/post_view/channel_intro_message/channel_intro_message.jsx

```diff
@@ -246,9 +247,12 @@ export function createDefaultIntroMessage(channel, centeredIntro, enableUserCrea
                     permissions={[Permissions.ADD_USER_TO_TEAM]}
                 >
                     {!teamIsGroupConstrained &&
-                    <span
+                    <ModalToggleButtonRedux
+                        id='introTextInvite'
                         className='intro-links color--link cursor--pointer'
-                        onClick={GlobalActions.showGetTeamInviteLinkModal}
+                        modalId={ModalIdentifiers.INVITATION}
+                        dialogType={InvitationModal}
+                        dialogProps={{}}
```

Comment:
Is `dialogProps` required? Just wondering if this will cause unnecessary renders.

Reply:
Actually not, there is a default value of `{}` in the component, so not passing it is the same thing. Good catch :)

Reply:
Done!